packages/babel-plugin-import-jsx-pragma/index.js

```diff
@@ -32,47 +35,77 @@ module.exports = function( babel ) {
 	function getOptions( state ) {
 		if ( ! state._options ) {
 			state._options = Object.assign( {}, DEFAULT_OPTIONS, state.opts );
+			if ( state._options.isDefault && state._options.scopeVariableFrag ) {
+				// eslint-disable-next-line no-console
+				console.warn( 'scopeVariableFrag is only available when isDefault is false' );
```

Comment:
Is it a necessary restriction? Or just to keep things simple? Seems like it wouldn't be terribly difficult to replace a JSXFragment node with a property access e.g. `<React.Fragment>`.

Reply:
This should not have been included, I'll remove it.

A bit of detail: by their nature, there can only be 1 default export from a given module. Therefore it made sense to keep the `isDefault` option for `scopeVariable`, but it will have no impact on `scopeVariableFrag` which will always be a named import. In theary, that could result in a configuration like:

```js
const webpackConfig = { // …
plugins: [
[
	require.resolve( '@wordpress/babel-plugin-import-jsx-pragma' ),
	{
		scopeVariable: 'React',
		scopeVariableFrag: 'Fragment',
		source: 'react',
		isDefault: true,
	},
],
[ require.resolve( '@babel/plugin-transform-react-jsx' ), {
	pragma: 'React.createElement',
	pragmaFrag: 'Fragment',
} ],
}
```

Admittedly contrived, but I hope that highlights my point 🙂